fdbserver/DataDistribution.actor.cpp

```diff
@@ -1375,6 +1393,9 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 			// Update server team information for consistency check in simulation
 			traceTeamCollectionInfo();
 		}
+
+		// Send a notification (about the SS team change) to one of the proxies.
+		Future<Void> notified = notifySSTeamChange(dbInfo);
```

Comment:
Can you add a placeholder of arguments for the actual change to `notifySSTeamChange`?